official/testing/docker_test.sh

```diff
@@ -31,16 +34,23 @@ EXIT=0
 # Get current directory path to mount
 export WORKSPACE=${PWD}
 
+if [ "$PYTHON_VERSION" = "PYTHON3" ]; then
+  DOCKER_IMG_NAME="tensorflow/tensorflow:1.3.0-py3"
+else
+  DOCKER_IMG_NAME="tensorflow/tensorflow:1.3.0"
+  if [ "$PYTHON_VERSION" != "PYTHON2" ]; then
+    echo "WARNING: Python version was not specified. Using Python2 by default."
+    sleep 5
```

Comment:
What's the reason to sleep here?

Reply:
So if you're running it manually you see the warning. A lot of times people will manually run our tests the warning will get lost.